src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -25,6 +30,7 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
         // TODO: add log message about successful fetched data from DB
```

Comment:
Let's remove unnecessary empty lines to keep the code clean (checklist: Don't forget about empty lines in the end of files).